src/utilities/generateAccountNumber/index.ts

```diff
@@ -0,0 +1,9 @@
+const generateAccountNumber = ():number => {
+    let accountNumber: string = ''
+    for (let i:number = 0; i < 8; i++) {
+        accountNumber += (Math.floor(Math.random() * (10)).toString())
+    }
+    return Number(accountNumber)
+}
+
+export default generateAccountNumber()
```

Comment:
Here you're exporting the result of calling the function: you want to just export the function name